packages/ocaml-monadic/ocaml-monadic.0.3.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "JHU PL Lab <pl.cs@jhu.edu>"
+authors: "JHU PL Lab <pl.cs@jhu.edu>"
+homepage: "https://github.com/zepalmer/ocaml-monadic"
+bug-reports: "https://github.com/zepalmer/ocaml-monadic/issues"
+license: "BSD-3-clause"
+dev-repo: "https://github.com/zepalmer/ocaml-monadic.git"
+build: [
+  ["oasis" "setup"]
```

Comment:
Ideally, you should include a `setup.ml` in your tarball (that means adding a tarball to your release where `oasis setup` was issued) so that the user does not need to have `oasis` installed.